neurolib/utils/adjust_params.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+def adjust_shape(original, target):
```

Comment:
This should not go into a separate file. @jajcayn: What about `utils/collections.py`? We have only the `dotdict` type in there.